sgkit/tests/test_aggregation.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import xarray as xr
+
+from sgkit.stats.aggregation import allele_count
+from sgkit.testing import simulate_genotype_call_dataset
+
+
+def get_dataset(calls, **kwargs):
+    calls = np.asarray(calls)
+    ds = simulate_genotype_call_dataset(
+        n_variant=calls.shape[0], n_sample=calls.shape[1], **kwargs
+    )
+    dims = ds["call/genotype"].dims
+    ds["call/genotype"] = xr.DataArray(calls, dims=dims)
+    ds["call/genotype_mask"] = xr.DataArray(calls < 0, dims=dims)
+    return ds
+
+
+def test_allele_count():
+    # Single-variant, single-sample case
+    ac = allele_count(get_dataset([[[1, 0]]]))
+    np.testing.assert_equal(ac, np.array([[1, 1]]))
+
+    # Multi-variant, single-sample case
```

Comment:
Can you break each of these tests into its own test function. They are independent, and the function name could then document them. E.g. `test_allele_count__single_variant_single_sample` (not sure if double underscore in the middle there is the standard way of delimiting the function name from extra description).

Reply:
https://github.com/pystatgen/sgkit/pull/36/commits/802c8d72dd1289ff4901db24755ff44ca51ec715#diff-9468bf42c32cc3634ef3f9386adc5bacR23